src/vs/workbench/api/node/extHost.api.impl.ts

```diff
@@ -409,6 +409,10 @@ export function createApiFactory(initData: IInitData, threadService: IThreadServ
 			onWillSaveTextDocument: (listener, thisArgs?, disposables?) => {
 				return extHostDocumentSaveParticipant.onWillSaveTextDocumentEvent(listener, thisArgs, disposables);
 			},
+			onDidChangeDiffInformation: (listener, thisArgs?, disposables?) => {
+				// TODO@joao
+				return createEmptyEvent()(listener, thisArgs, disposables);
```

Comment:
We could the be smart about this and *only* send these event from the main side when we have listeners on the ext host side